moveit_core/trajectory_processing/CMakeLists.txt

```diff
@@ -44,19 +44,8 @@ if(BUILD_TESTING)
   ament_add_gtest(test_time_parameterization test/test_time_parameterization.cpp
     APPEND_LIBRARY_DIRS "${append_library_dirs}")
 
-  target_include_directories(test_time_parameterization PUBLIC
-    ${geometric_shapes_INCLUDE_DIRS}
-  )
-
   target_link_libraries(test_time_parameterization
     moveit_test_utils
-    moveit_robot_trajectory
-    moveit_trajectory_processing
-    ${urdfdom_LIBRARIES}
-    ${srdfdom_LIBRARIES}
-    ${urdfdom_headers_LIBRARIES}
     ${MOVEIT_LIB_NAME}
-    ${geometric_shapes_LIBRARIES}
-    resource_retriever::resource_retriever
   )
```

Comment:
`robot_state`, `robot_model`, and `robot_trajectory` are included but not listed here.

Reply:
same